docs/content/docs/data-structures/overview.md

```diff
@@ -86,11 +86,12 @@ The SharedString DDS is used for unstructured text data that can be collaborativ
 
 ## Specialized data structures
 
-- [SharedCounter][] -- a counter.
+- [SharedCounter][{{< relref "counter.md" >}}] -- a counter.
 
-  The SharedCounter is useful to keep track of increments. While a key-value data structure appears like a good fit, two
-  clients simultaneously setting the same key can cause issues. By contrast, clients can increase or decrease the
-  SharedCounter value by a specified amount, but they can't set it to a specified value. It is optimistic.
+  The `SharedCounter` is useful to keep track of increments.
+  While a key-value data structure appears like a good fit, two clients simultaneously setting the same key can cause issues.
+  By contrast, clients can increase or decrease the `SharedCounter` value by a specified amount, but they can't set it to a specified value.
```

Comment:
This is more an allusion to the problem than an explanation. Can we elaborate about this and include a concrete example?

Reply:
I added an explanation + example to the SharedCounter article and linked to it from here.